pkg/template/template.go

```diff
@@ -94,3 +108,36 @@ func mapType(t Type) prometheus.ValueType {
 		return prometheus.UntypedValue
 	}
 }
+
+func (m *MetricTemplate) sendMergedMetric(vals []float64, labels [][]string, ch chan<- prometheus.Metric) {
+	l := make(map[string]string)
+	l["customizations"] = strings.Replace(m.Name, fmt.Sprintf("%s_", metricShootsCustomPrefix), "", 1)
```

Comment:
```suggestion
	l["customization"] = strings.Replace(m.Name, fmt.Sprintf("%s_", metricShootsCustomPrefix), "", 1)
```
change the label to be singular